src/router.ts

```diff
@@ -319,23 +323,13 @@ router.beforeEach((to, from, next) => {
   next()
 })
 
-function getTitleMeta(route) {
-  let meta
-  for (let index = route.matched.length - 1; index >= 0; index -= 1) {
-    meta = route.matched[index].meta
-    if (meta.headTitle) {
-      if (typeof meta.headTitle === "function") {
-        return meta.headTitle(route.params)
-      }
-      return meta.headTitle
-    }
-  }
-  return process.env.VITE_TITLE
-}
-
 router.afterEach((to) => {
   nextTick(function () {
-    document.title = getTitleMeta(to)
+    if (!to.hash) {
```

Comment:
Proposition pour que le to.hash soit encore plus explicit : https://github.com/betagouv/aides-jeunes/commit/95571b07f889a99d112f86f478d6635942af45ba

Reply:
Oui, oui, merci beaucoup !!